core/src/main/resources/xsl/authenticated-by.xsl

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  authenticated-by.xsl
+
+  This stylesheet is responsible for transforming rax:authenticatedBy
+  found in resource or method attributes in a WADL into header params.
+  Header params are always placed at the request level.
+
+  The header checks are instructed to always return a 403 with the
+  message: "You are forbidden to perform the operation"
+
+  Copyright 2014 Rackspace US, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:wadl="http://wadl.dev.java.net/2009/02"
+                xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+                xmlns:rax="http://docs.rackspace.com/api">
+    <xsl:template match="node() | @*">
+        <xsl:copy>
+            <xsl:apply-templates select="@* | node()"/>
+        </xsl:copy>
+    </xsl:template>
+
+    <!--
+        Specifying assertions on the X-AUTHENTICATED-BY request header with different
+        casing (e.g. X-Authenticated-BY) will completely disable the check, so don't allow it.
+    -->
+    <xsl:template match="wadl:param[upper-case(@name)='X-AUTHENTICATED-BY' and @style='header' and not(ancestor::*/wadl:response)]">
+        <xsl:message>[WARNING] You are not allowed to specify an X-AUTHENTICATED-BY header request parameter.</xsl:message>
+        <xsl:message>[WARNING] The X-AUTHENTICATED-BY header parameter will be ignored.</xsl:message>
+    </xsl:template>
```

Comment:
Oh man, that's a bug...and a really big one...the casing in the header name shouldn't matter.   Can you do me a favor?  Do a manual test to confirm the bug with the wadltest CLI tool. I'm wondering if this is actually an issue with the test framework and not an actual bug.

If it is an actual bug...copy pasta a WADL that illustrates it in a new github issue, and I'll get on that right away.